packages/web-components/fast-components/src/date-picker/date-picker.styles.ts

```diff
@@ -0,0 +1,161 @@
+import { css, ElementStyles } from "@microsoft/fast-element";
+import {
+    display,
+    ElementDefinitionContext,
+    FoundationElementDefinition,
+} from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import { heightNumber } from "../styles/index";
+import {
+    accentForegroundActive,
+    bodyFont,
+    controlCornerRadius,
+    designUnit,
+    disabledOpacity,
+    fillColor,
+    foregroundOnAccentActive,
+    neutralFillRest,
+    neutralForegroundRest,
+    strokeWidth,
+    typeRampBaseFontSize,
+    typeRampBaseLineHeight,
+    typeRampPlus3FontSize,
+    typeRampPlus3LineHeight,
+} from "../design-tokens";
+import { elevation } from "../styles/elevation";
+import { DirectionalStyleSheetBehavior } from "../styles";
+
+export const datePickerStyles: (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => ElementStyles = (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => css`
+    :host {
+        --elevation: 14;
+        --panel-size: 240px;
```

Comment:
Ideally we can base the overall size on the padding and margin applied to individual components. This will help with adaptive density and type ramp support.

Reply:
Calendar doesn't work off of padding and margin. I thought it would be more useful to scale to it's parent container or be sized to a specific width and just use grid columns to adjust. I was also worried about trying to calculate widths of a 7 even column layout, a 4 even column layout for month and year pickers and the time picker layout with the separator and 3 even columns. Setting a fixed width solves these complications.

I've pushed a change that changes this width with a calculation based on a cell width for the calendar. I built in some flex for density.